pages/windows/test-netconnection.md

```diff
@@ -0,0 +1,13 @@
+# Test-NetConnection
+
+> Displays diagnostic information for a connection.
```

Comment:
[Please use the imperative in descriptions](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#imperative-mood).
```suggestion
> Display diagnostic information for a connection.
```